RevenueCatUI/Templates/TemplateViewType.swift

```diff
@@ -1,57 +1,63 @@
 import RevenueCat
 import SwiftUI
 
+/// A `SwiftUI` view that can display a paywall with `TemplateViewConfiguration`.
 @available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
 protocol TemplateViewType: SwiftUI.View {
 
-    init(
-        packages: [Package],
-        localization: PaywallData.LocalizedConfiguration,
-        paywall: PaywallData,
-        colors: PaywallData.Configuration.Colors
-    )
+    init(_ configuration: TemplateViewConfiguration)
+
+}
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+private extension PaywallTemplate {
+
+    var packageSetting: TemplateViewConfiguration.PackageSetting {
+        switch self {
+        case .example1: return .single
```

Comment:
Very simple `switch` from `PaywallTemplate` to `TemplateViewConfiguration.PackageSetting`.
When we add a new template, this makes it obvious that we need to specify this setting.